src/com/fundation/search/common/Search.java

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
Christian, please don't forget update your new code for review

Reply:
Yes I update my code now